server/events/models/models.go

```diff
@@ -522,6 +524,20 @@ func (c CommandName) String() string {
 		return "apply"
 	case PlanCommand:
 		return "plan"
+	case UnlockCommand:
+		return "unlock"
 	}
 	return ""
 }
+
+// DeleteLockCommandResult is the result of attempting to delete an atlantis lock.
+type DeleteLockCommandResult int
```

Comment:
I've made some changes that make this unnecessary, mainly following the pattern before where `error` means error and `nil` means lock not found.